src/main/shadow/cljs/ui/main.cljs

```diff
@@ -8,7 +8,7 @@
     [shadow.cljs.model :as m]
     [shadow.cljs.ui.components.inspect :as inspect]
     [shadow.cljs.ui.components.dashboard :as dashboard]
-    [shadow.cljs.ui.components.runtimes :as runtimes]
+    [shadow.cljs.ui.views.runtimes :as runtimes]
```

Comment:
Why not modify the `components` file? Please don't introduce `views` which is just another name for `components`. I don't like `views`.

Reply:
Since this was the first part I thought maybe it will be easier to have another ns. Will follow with components.